components/shared/components/Widget/components/panes/PaymentPane/PaymentPane.tsx

```diff
@@ -40,22 +35,23 @@ export const PaymentPane: React.FC<{
             pane3_referrals_title: text.pane3_referrals_title,
           }}
         />
-      )}
-      {method === PaymentMethod.VIPPS && (
+      );
+    }
+    case PaymentMethod.VIPPS: {
+      if (!vipps) {
+        throw new Error("Missing configuration for Vipps, but selected payment method is vipps");
```

Comment:
What happens when this is thrown?
How is it handled?

Reply:
No handling, it would be unexpected though. It means that the user has somehow been able to select vipps payment method, but there is no configuration for vipps. I'm not sure if there are better ways of dealing with it?